docs/arf.md

```diff
@@ -1679,21 +1679,12 @@ Wallet Provider.
 
 A User installs a Wallet Instance because they want to obtain and use
 one or more PIDs and/or attestations. However, there is no requirement
-for a PID Provider or an Attestation Provider to support all certified
+for a PID Provider to support all certified
 Wallet Solutions in the EUDI Wallet ecosystem. This means that the PID
-Provider or an Attestation Provider are not obliged to issue a PID or an
-attestation respectively to any certified Wallet Instance upon the
+Providers are not obliged to issue a PID to any certified Wallet Instance upon the
```

Comment:
why remove the statement that attestation provider does not have to support all wallets? the proposed suggestion might raise questions "are attestation providers obliged to support all wallets?"

Reply:
that's my point, Attestation Providers must support all eIDAS Wallets